apps/taxes/src/modules/avatax/ui/form-helper-text.tsx

```diff
@@ -0,0 +1,19 @@
+import { Text } from "@saleor/macaw-ui/next";
+import React from "react";
+
+export const HelperText = ({
+  children,
+  disabled = false,
+}: {
+  children: React.ReactNode;
+  disabled?: boolean;
+}) => {
+  return (
+    <Text
+      color={disabled ? "textNeutralDisabled" : "textNeutralSubdued"}
+      fontWeight={"captionLarge"}
```

Comment:
why not `variant="caption"`?

Reply:
It doesn't change the color on `disabled`. This is the only combination I found that has the same size as the link that is displayed alongside it and has the right colors.